### ISSUE TITLE
What is the relationship between `AmbientColor::brightness` and Blender's background color strength?

### ISSUE THREAD

**Issue by janhohenheim** [bevyengine/bevy#12280]
During Bevy 0.13's development, lots of attention went to `PointLight`, `DirectionalLight` and `Spotlight`. However, I failed to find any significant discussion of `AmbientLight`, neither on GitHub nor Discord. I'd be glad to be proved wrong if anyone's got a link though :)

The reason I care about this because the general consensus has been "let's try to have some kind of conformance with Blender", using Blender's EV100 and render outputs as direct references for how Bevy should behave. This worked out (mostly, there's still talk about Blender itself being wrong in places) for the lights mentioned, but `AmbientLight` kind of got left behind.

In Blender, under the tab "World", you can find the background light (aka environment light). The meaning behind its `strength` param is a bit elusive to me, and [the docs are not helping](https://docs.blender.org/manual/en/latest/render/shader_nodes/shader/background.html).

This is how Blender renders a scene with no lights and only a background light of strength 1:

![image](https://github.com/bevyengine/bevy/assets/9047632/00ca90de-016f-4ff7-8fbe-3b2f04d03fb0)

This is how Bevy renders the same scene with no lights and an `AmbientLight` of the same color with brightness 1:
![image](https://github.com/bevyengine/bevy/assets/9047632/eb0830e1-826d-44e1-bf66-745be6549c70)

As you can see, Bevy is *much* darker. (Ignore the procedural sky)

Let's bump the brightness to the [slightly magic constant 683](https://github.com/search?q=repo%3AKhronosGroup%2FglTF-Blender-IO%20PBR_WATTS_TO_LUMENS%20&type=code):
![image](https://github.com/bevyengine/bevy/assets/9047632/3687820d-de71-461c-87a4-a1ad8b76acb5)

Better, but very blue.

https://github.com/google/filament/issues/1667 says that using Blender's exposure settings (which are Bevy's standard settings), the background strength should be the same as their IBL intensity, which I believe corresponds to Bevy's [`EnvironmentMapLight::intensity`](https://docs.rs/bevy/latest/bevy/pbr/prelude/struct.EnvironmentMapLight.html#structfield.intensity). I would love to test this, but I fail to create a valid environment map from a simple `Color` using `Image::new` since I keep getting an error like "Texture binding 13 expects dimension = Cube, but given a view with dimension = D(1, 2 or 3)" no matter what I do. I am not an expert in the `Image` API, so this is probably possible in some way.

**Comment by fintelia** [comment]
Related: #11933

**Comment by janhohenheim** [comment]
@superdump do you have some thoughts on this? I'm not expecting a full answer, I'd just like to know what you think about my hunch.

**Comment by superdump** [comment]
So I think @fintelia ’s answer here: https://github.com/bevyengine/bevy/issues/11933#issuecomment-1995427587 defines the physical units. Basically the ambient colour multiplied by the ambient brightness gives luminance in cd/m^2. Then exposure is applied to that. As for how to match that in blender, I don’t know without digging deeper.

**Comment by janhohenheim** [comment]
@superdump am I right in my assumption that this work differently when using a monochrome `EnvironmentLightMap`? 
If so, I'd like to test if this is the same behavior as in Blender. Do you have some tips on how I could construct an `Image` for such a lightmap? Every time I tried to create an image manually for it, I ran into problems like "Texture binding 13 expects dimension = Cube, but given a view with dimension = D(1, 2 or 3)"

**Comment by superdump** [comment]
Yup, you need a cubemap texture, which is a texture with 6 cube faces ordered in bevy conventions like looking along +x, -x, +y, -y, -z, +z. Note that the order of z is swapped to be minus first then plus. This is because cubemap textures in graphics APIs are left-handed. Left-handed y-up has positive z going into the screen. Right-handed y-up that bevy uses has positive z coming out of the screen. Hence we need to swap the order of the z faces.

The skybox example uses cubemap textures and specifically a PNG that doesn’t have the metadata to communicate to the asset loaders that it is a cubemap, where the ktx2 textures do. You can see how that bit is handled here: https://github.com/bevyengine/bevy/blob/85b488b73d6f6e75690962fba67a144d9beb6b88/examples/3d/skybox.rs#L152 . If you take a look at the PNG you can also see how they’re laid out. I think the faces are stacked vertically, and the faces are square, so we divide the height by the width to know how many faces there are.

**Comment by janhohenheim** [comment]
Update: it seems like using the environment map leads to more desirable effects.

Blender:
![image](https://github.com/bevyengine/bevy/assets/9047632/2d5cbf66-70ce-4ac0-8c34-46a64e013415)

AmbientLight = background color * background strength * 400:
![image](https://github.com/bevyengine/bevy/assets/9047632/738f0179-cd59-4b1d-a43c-34533ced141e)

No AmbientLight, but a lightmap with 6 sides, each containing just one pixel = background color * background strength * 400:
![image](https://github.com/bevyengine/bevy/assets/9047632/9e2bdfcc-8792-4cbd-8925-717cba7a4c6a)

Everything is using Agx. It's weird to me that I needed to multiply the strength by a big constant, otherwise everything is still very dark. Bevy also seems less saturated than Blender for some reason.

**Comment by janhohenheim** [comment]
Labeling as `D-Modest` as no one I ever asked about this was able to answer, so I suppose this actually requires a bit of detective work.
